docs/src/Upsell.doc.js

```diff
@@ -1,16 +1,30 @@
 // @flow strict
 import React, { type Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Upsell"
-    description="Upsells are banners that display short messages that focus on selling or upgrading something the user already has."
+    description="Upsells are banners that display short messages that focus on promoting an action or upgrading something the user already has."
+    defaultCode={`
+    <Upsell
+      title="Give $30, get $60 in ads credit"
+      message="Earn $60 of ads credit, and give $30 of ads credit to a friend"
+      primaryAction={{href: "https://pinterest.com", label: "Send invite"}}
```

Comment:
nit: Can we add target blank?
`      primaryAction={{href: "https://pinterest.com", target: 'blank', label: "Send invite"}}
`

Reply:
Same for most of all examples.